hugo/assets/js/clipboard.js

```diff
@@ -0,0 +1,56 @@
+// Adapted from https://logfetch.com/hugo-add-copy-to-clipboard-button/
+const svgCopy =
```

Comment:
As a general remark, it may be worthwhile to try and reduce the amount of DOM manipulation in this snippet.
For example, you could include in the shortcode:
```
<button>
    <img src="assets/copy.svg">
</button>
<button>
    <img src="assets/paste.svg">
</button>
```

Reply:
Agreed, I've done that now in commit 802f3b94709a2f89749b2fc447e4150b5fb7508.